README.md

```diff
@@ -147,6 +161,31 @@ http:
           forwardedHeadersCustomName: X-Custom-Header
           redisCacheEnabled: false
           redisCacheHost: "redis:6379"
+          crowdsecLapiTlsCertificateAuthority: |-
+            -----BEGIN CERTIFICATE-----
```

Comment:
put this in a exemple docker-compose, this is not a default value

Reply:
it's not default, it is an example of values

Like above

```yaml
          forwardedHeadersCustomName: X-Custom-Header
```

Reply:
but it take to much space ...